extensions/presentation.sigmf-ext.md

```diff
@@ -0,0 +1,239 @@
+# The `presentation` SigMF Extension Namespace v1.0.0
+
+This document defines the `presentation` extension namespace for the Signal
+Metadata Format (SigMF) specification. This extension namespace contains objects
+to help define how SigMF metadata should be presented graphically to users.
+
+All fields in the `presentation` extension are optional, thus any SigMF
+application using this extension is not required to implement any particular
+feature. It is RECOMMENDED that applications adhering to the `presentation`
+extension use default values in this document. It is also RECOMMENDED that
+applications implementing this extension describe what fields are implemented.
+
+## 0 Datatypes
+
+The `presentation` extension defines the following SigMF datatypes:
+
+|type|long-form name|description|
+|----|--------------|-----------|
+|`color`|Hexadecimal Color String|String representing a 24 bit color (with optional alpha value) in hexadecimal form; either "#RRGGBB" or "#AARRGGBB".|
+
+### 0.1 The `color` datatype
+
+The `color` datatype is a string which MUST be either 6 or 8 hexadecimal
+characters long prepended by a "#". It MAY be specified in capital or lower case
+characters and applications MUST handle either. The alpha channel is optional,
+and while compliant software MUST be able to parse either format presentation of
+the alpha channel by visualization software is RECOMMENDED but OPTIONAL. If
+software is unable to render the alpha channel then it SHOULD ignore the alpha
+value for foreground objects and ignore background objects entirely.
+
+ARGB format was selected over RGBA for compatibility with QColor objects, RGBA
+color specification is invalid and will result in incorrect display of content.
+
+For greatest compatibility, it is RECOMMENDED that this value default to
+`#00ffffff` (fully opaque white) for foreground features and `#ffffffff` (fully
```

Comment:
I think you might have your alpha's switched on the two color codes used in this one line.

Reply:
Yep, thanks. Fixed.